components/fxa-client/src/lib.rs

```diff
@@ -2,597 +2,1268 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+//! # Firefox Accounts Client
```

Comment:
Careful, this large diff is not rendered by default! :-)

This file is in fact brand new, and git's diff view is not going to help you here. I recommend just viewing it as a separate file.

This is the public API of the crate, from which we derived the `.udl` file and hence from which the Kotlin and Swift bindings are generated.